test/unit/models/flag.test.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable security/detect-object-injection */
+const chai = require("chai");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+
+const flagModel = require("../../../models/flag");
+const firestore = require("../../../utils/firestore");
+const flagFirestore = firestore.collection("featureFlags");
+const flagData = require("../../fixtures/flag/flag")();
+
+describe("flags", function () {
+  let flagId;
+
+  beforeEach(async function () {
+    flagId = await flagModel.addFlag(flagData);
+  });
+
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("add feature flag", function () {
+    it("Should return flag Id", async function () {
+      const fetched = await flagFirestore.doc(flagId).get();
```

Comment:
NIT: `flagSnapshot` will be a better variable name

Reply:
will do. By the way what is `NIT`?

Reply:
> will do. By the way what is `NIT`?

NIT in this context means nit pick. Its generally used for small change requests and suggestions.